SGHTTPRequest/Core/SGHTTPRequest.h

```diff
@@ -367,6 +368,10 @@ a new identical request.
  */
 - (void)removeCacheFilesIfExpired;
 
+#pragma mark Network Stubbing Data Source Injection
 
+@property (class, nonatomic, assign) id<SGNetworkStubsDataSource>_Nullable networkStubsDataSource;
 
++ (void)setNetworkStubsDataSource:(id<SGNetworkStubsDataSource>_Nullable)networkStubsDataSource;
```

Comment:
We shouldn't need to explicitly specify 
`+ (void)setNetworkStubsDataSource:(id<SGNetworkStubsDataSource>_Nullable)networkStubsDataSource;`
or
`+ (id<SGNetworkStubsDataSource>_Nullable)networkStubsDataSource;`

 The `@property` should be sufficient.